components/builder-web/app/shared/github-repo-picker/github-repo-picker.component.ts

```diff
@@ -36,19 +35,19 @@ export class GitHubRepoPickerComponent implements OnInit {
   repoSelectCancel: Function;
 
   constructor(private store: AppStore) {
-    this.store.dispatch(fetchGitHubOrgs());
+    // this.store.dispatch(fetchGitHubOrgs());
```

Comment:
Commented lines here and below are intentional; the actions they refer to are removed in this PR, but these components are not currently in use.